src/constants/sizes.ts

```diff
@@ -4,9 +4,11 @@ export const BASE_SIZES = {
   LARGE: "large"
 } as const;
 
+export const BASE_SIZES_WITH_NONE = { NONE: "none", ...BASE_SIZES };
+
 export const SIZES = { XXS: "xxs", XS: "xs", ...BASE_SIZES } as const;
 
-export const DialogPositions = {
+export const DialogPosition = {
```

Comment:
was not sure about it because the standard we have for enums was singular naming but we did not convert it while converting other components so I did no convert it to enum...
you think it will be better to convert it to regular enum?